content/admin/policy_mgmt/SecurityControls.md

```diff
@@ -1,81 +1,74 @@
+If Contrast recognizes potential security controls, they're automatically populated in a **Suggestions** section in the Security Controls grid. (You can hide the section by clicking on the carrot in the header row.) Suggestions are governed by [Enterprise Access Control](admin-manageorgs.html#access), which allows a user to see suggestions from applications only if they have a **View** role or higher.
 
-Clicking the **Delete** (trashcan) icon will remove the suggestion but provide the option to Undo.  There are no repeat suggestions, so once deleted, an API will never be suggested again. As of now, there is no way to view historical suggestions or get them back.
+Contrast names suggested security controls with the class and method; however, you can edit the Name, API and Type fields inline before adding it as an active security control. Hovering over the API gives the name of each application in which it was discovered, which links directly to the Vulnerabilities tab in the application's Details page. 
 
-<a href="assets/images/SecurityControlUndoRemove.png" rel="lightbox" title="Undo Option When Removing A Suggested Security Control"><img class="thumbnail" src="assets/images/SecurityControlUndoRemove.png"/></a>
+<a href="assets/images/Security-controls-suggested.png" rel="lightbox" title="Suggested security controls"><img class="thumbnail" src="assets/images/Security-controls-suggested.png"/></a>
 
+Click the **Add** icon in the row to accepts the suggestion and move it up in the grid as a Rule. Anyone with a **Rules_Admin** role can then view or edit it. Click the **Delete** icon to remove the suggestion (with option to Undo it). Contrast doesn't repeat suggestions, so once it's deleted, an API is never suggested again. There is no way to view historical suggestions or get them back.
 
 ## Notifications
 
-Auto-discovered security controls will generate notifications OnEvent in Contrast the very first time they are discovered, regardless of application. 
-If a control is discovered for an app that a user doesn't have *View* role or higher for, they will not receive the notification.
+Auto-discovered security controls generate notifications OnEvent in Contrast the very first time they are discovered, regardless of the application. If a control is discovered for an application that for which a user doesn't have **View** role or higher, they won't receive the notification.
```

Comment:
"If a control is discovered for an application for which a user doesn't..." (remove the word "that")